UI/window-missing-files.hpp

```diff
@@ -21,7 +21,7 @@
 #include <QStyledItemDelegate>
 #include "obs-app.hpp"
 #include "ui_OBSMissingFiles.h"
-
+#include <QCloseEvent>
```

Comment:
If there was a blank line here before, it should be retained.  This include should be grouped with the other Qt includes.